modules/identity-provider-registering-github.adoc

```diff
@@ -17,19 +17,20 @@ https://github.com/settings/applications/new[Register a new OAuth application].
 ** For GitHub Enterprise, go to your GitHub Enterprise home page and then click
 *Settings -> Developer settings -> Register a new application*.
 . Enter an application name, for example `My OpenShift Install`.
-. Enter a homepage URL, such as `https://myapiserver.com:8443`.
+. Enter a homepage URL, such as 
+`https://console-openshift-console.apps.example-openshift-cluster.com/`.
 . Optionally, enter an application description.
 . Enter the authorization callback URL, where the end of the URL contains the
 identity provider `name`:
 +
 ----
-<apiserver>/oauth2callback/<identityProviderName>
+https://openshift-authentication-openshift-authentication.apps.<clusterDomain>/oauth2callback/<identityProviderName>/
 ----
 +
 For example:
 +
 ----
-https://myapiserver.com:8443/oauth2callback/github/
+https://openshift-authentication-openshift-authentication.apps.example-openshift-cluster.com/oauth2callback/github/
```

Comment:
Then also need update as above comment.

Reply:
I've used the above format, but kept it as an explicit URL instead of using replaceables.